index.rst

```diff
@@ -121,6 +121,115 @@ BPS maintains state in the local filesystem for its preparation, submission, rep
+We will allow the use of either deterministic or non-deterministic RSEs in all cases, with deterministic preferred in order to minimize the size of the Rucio database.
+
+The deterministic RSE algorithm for raw and simulated data can be ``identity``, which includes the Scope (see next subsection) in the physical filename, or ``lsst_butler``, which does not, at a site's option.
+Raw and simulated data will be distinguished by instrument, even if Scope is not used.
+
+The deterministic RSE algorithm for Butler data products will be ``identity`` so that the Scope is included.
+
+Scopes
+------
+
+Scopes are useful for constraining access permissions and may be useful for controlling backups and removing unneeded or obsolete data en masse.
+There is no significant limit on the number of Scopes, nor on the number of DIDs per Scope.
+
+raw
+^^^
+Represents raw archival observation data from any instrument, whether on-sky, calibration, engineering, or on a test stand.
+Includes precursor on-sky data, if desired.
+In any case, includes data from multiple instruments, such as the three Rubin Observatory ones.
+Manipulations of this Scope need to be highly restricted.
+
+Raw data can be stored in a different location from Butler data products for safety, and it will typically be ingested into Butler repos via the ``direct`` method.
+
+simulation
+^^^^^^^^^^
+Manipulations of this Scope are less restricted than for raw data.
+
+Campaign Scope
+^^^^^^^^^^^^^^
+The Scopes for campaigns will follow Butler repo naming: each repo will be a Scope.
+This means that it will not be possible to use ``/repo/main`` at the USDF and a different repo name at the UKDF and FrDF.
+
+Each campaign (e.g. DP1, DP2, DR1, DR2, etc. or preparatory campaigns) can have its own Butler repo and therefore Scope for data products.
+But repos can also be shared across multiple campaigns if desired.
+
+Manipulations of campaign Scopes are inherently multi-site and open to a wider group.
+
+All Butler repos used for multi-site operations will be expected to be physically available under a single root path (via the use of symlinks if necessary on POSIX filesystems).
+
+Containers
+----------
+
+raw and simulation Scopes
+^^^^^^^^^^^^^^^^^^^^^^^^^
+Not expected to be used.
+
+Campaign Scope
+^^^^^^^^^^^^^^
+Each step will have all of its outputs collected into a single Container for ease of archiving to a Tape RSE.
+The name will be of the form ``Container/{campaign}/{step}``.
+These Containers can be closed when the step is complete.
+
+Datasets
+--------
+
+raw and simulation Scope
+^^^^^^^^^^^^^^^^^^^^^^^^
+Datasets are named as ``Dataset/{instrument}/raw/TractNNNNN``, as there are not too many files per tract.
+For LSSTCam, we expect to have on average 120000 files per tract, more in deep drilling fields.
+If necessary, to keep Dataset sizes small, a ``/YYYY`` year field could be appended to the Dataset name.
```

Comment:
My [understanding](https://rucio.cern.ch/documentation/started/concepts/file_dataset_container/#datasetcontainer-status) is that Datasets can be added to and, if non-monotonic, can even have content removed.  If we don't have a YYYY component, we won't be able to close the raw Dataset, though.

Having a YYYY based on the `day_obs` is the easiest, but having it correspond to the Data Release might be even better.  Since the DRs are not set by the calendar, this would involve some configuration of the Dataset generation tooling.  I'm not sure if the complexity is worth it.